### PR TITLE
valgrind: added suppression for cython constants

### PR DESCRIPTION
**Pull request by dillaman**
Signed-off-by: Jason Dillaman <dillaman@redhat.com>

### CODE REVIEW

**Comment by dillaman** [comment]
Fixes false positives in test runs: http://qa-proxy.ceph.com/teuthology/teuthology-2017-12-08_02:01:03-rbd-master-distro-basic-smithi/1943792/teuthology.log

File: teuthology/task/install/valgrind.supp [hunk 1]
```diff
@@ -445,3 +445,41 @@
     ...
     fun:PyObject_Call
 }
+
+# rados cython constants
+{
+   <insert_a_suppression_name_here>
```
Comment (trociny) [inline teuthology/task/install/valgrind.supp]: @dillaman probably insert a suppression name here as suggested or remove this line (and below)?

File: teuthology/task/install/valgrind.supp [hunk 2]
```diff
@@ -445,3 +445,41 @@
     ...
     fun:PyObject_Call
 }
+
+# rados cython constants
```
Comment (trociny) [inline teuthology/task/install/valgrind.supp]: could be used as a suppression name

**Comment by dillaman** [comment]
@trociny addressed

**Pull request merged**